src/components/course/course-header/CourseHeader.jsx

```diff
@@ -35,6 +35,7 @@ import {
   useEnterpriseCustomerContainsContent,
   useIsAssignmentsOnlyLearner,
 } from '../../app/data';
+import usePassLearnerCsodParams from '../../app/data/hooks/usePassLearnerCsodParams';
```

Comment:
nit: To keep import paths clean, it's recommended to include this import in the `'../../app/data'` group above. To support that, any hook created within `../../app/data/hooks/` should also be added [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/src/components/app/data/hooks/index.js) to support the barreling similar to other hooks in that directory (see https://flaming.codes/en/posts/barrel-files-in-javascript/).

Reply:
Thank you @adamstankiewicz for these great suggestions. I'll create a follow-up PR to address them.